docs/spec/staking/state.md

```diff
@@ -1,204 +1,148 @@
-
 ## State
 
-The staking module persists the following information to the store:
-* `GlobalState`, a struct describing the global pools, inflation, and
-  fees
-* `ValidatorCandidates: <pubkey | shares> => <candidate>`, a map of all candidates (including current validators) in the store,
-indexed by their public key and shares in the global pool.
-* `DelegatorBonds: < delegator-address | candidate-pubkey > => <delegator-bond>`. a map of all delegations by a delegator to a candidate,
-indexed by delegator address and candidate pubkey.
-  public key
-* `UnbondQueue`, the queue of unbonding delegations
-* `RedelegateQueue`, the queue of re-delegations
-
-### Global State
-
-The GlobalState contains information about the total amount of Atoms, the
-global bonded/unbonded position, the Atom inflation rate, and the fees.
-
-`Params` is global data structure that stores system parameters and defines overall functioning of the 
-module.
-
-``` go
-type GlobalState struct {
-    TotalSupply              int64        // total supply of Atoms
-    BondedPool               int64        // reserve of bonded tokens
-    BondedShares             rational.Rat // sum of all shares distributed for the BondedPool
-    UnbondedPool             int64        // reserve of unbonding tokens held with candidates
-    UnbondedShares           rational.Rat // sum of all shares distributed for the UnbondedPool
-    InflationLastTime        int64        // timestamp of last processing of inflation
-    Inflation                rational.Rat // current annual inflation rate
-    DateLastCommissionReset  int64        // unix timestamp for last commission accounting reset
-    FeePool                  coin.Coins   // fee pool for all the fee shares which have already been distributed
-    ReservePool              coin.Coins   // pool of reserve taxes collected on all fees for governance use
-    Adjustment               rational.Rat // Adjustment factor for calculating global fee accum
-}
+### Pool
+ - index: n/a single-record
```

Comment:
What is this referring to? Is this removed by the parser?

Reply:
see above comment